Documentation: add reference to required settings to ingressroute docs (#6527)
gethomepage/homepage: docs/configs/kubernetes.md

### Traefik IngressRoute support

> Once you've set `traefik: true` in `kubernetes.yaml`, homepage can also read ingresses defined using the Traefik IngressRoute custom resource definition. Due to the complex nature of Traefik routing rules, it is required for the `gethomepage.dev/href` annotation to be set:

Reviewer: ```suggestion
If enabled (with `traefik: true` in kubernetes.yaml), homepage can also read ingresses defined using the Traefik IngressRoute custom resource definition. Due to the complex nature of Traefik routing rules, it is required for the `gethomepage.dev/href` annotation to be set:
```
